src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/IUIScrollView.iOSmacOS.cs

```diff
@@ -13,10 +13,14 @@ internal interface IUIScrollView
 	{
 		CGPoint UpperScrollLimit { get; }
 
-		void SetContentOffset(CGPoint contentOffset, bool animated);
+		CGPoint ContentOffset { get; }
+
+		void ApplyContentOffset(CGPoint contentOffset, bool animated);
 
 #if __IOS__
-		void SetZoomScale(nfloat scale, bool animated);
+		nfloat ZoomScale { get; }
+
+		void ApplyZoomScale(nfloat scale, bool animated);
```

Comment:
There is no need to change it right now, but I do think that we should really follow the UWP/WinUI logic to set all values (offsets and zoom) at once in a single method, this avoids weird sync issues. For instance what's the behavior if change both with the animated flag?

Reply:
Will create separate issue for it 👍

Reply:
If you want, but actually it's more a general guideline while working in the SV, than a real issue. I learned it the hard way when I had to do a massive refactor due to that. But personnaly I won't even bother to open a specific issue for that.